### PR TITLE
📖story-ads: update docs

### PR DESCRIPTION
**Pull request by calebcordry**
Trying to get this in before launch. Better docs cleanup coming this week.

### CODE REVIEW

File: extensions/amp-story/amp-story-auto-ads.md [hunk 1]
```diff
@@ -75,45 +72,28 @@ looks like the following:
...
+respected by the insertion algorithm.
 
-    <template type="amp-mustache" id="template-2">
-      <div class="creative-line-1">{{creativeLine1}}</div>
-      <div class="creative-line-2">{{creativeLine2}}</div>
-      <amp-pixel src="{{impressionUrl}}"></amp-pixel>
-    </template>
-  </amp-story-auto-ads>
+```html
+<amp-story-page next-page-no-ad id="page-2"
```
Comment (torch2424) [inline extensions/amp-story/amp-story-auto-ads.md]: Missing `>`

File: extensions/amp-story/amp-story-auto-ads.md [hunk 2]
```diff
@@ -75,45 +72,28 @@ looks like the following:
...
-    </script>
+It has to be a direct child of `amp-story` element.
 
-    <template type="amp-mustache" id="template-1">
-      <amp-img layout="fill" src="{{imgSrc}}"></amp-img>
-      <amp-pixel src="{{impressionUrl}}"></amp-pixel>
-    </template>
+## Insertion Control
+The AMP runtime will determine the best place to insert an ad using our own heuristics.
+If there is a specific position in a story that you wish to never show an ad, you
```
Comment (torch2424) [inline extensions/amp-story/amp-story-auto-ads.md]: nit: I think this is worded weird (at least to me). Can we do something like: 


> If there is a specific position in a story that you wish to never show an ad, you can add the `next-page-no-ad` attribute to the `<amp-story-page>`. The insertion algorithm will then ignore the page following these `<amp-story-page>` elements as possible places to place an ad.

Comment (torch2424) [reply extensions/amp-story/amp-story-auto-ads.md]: If that sounds too redundant, feel free to just ignore this.

Comment (calebcordry) [reply extensions/amp-story/amp-story-auto-ads.md]: made changes.

File: extensions/amp-story/amp-story-auto-ads.md (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-story/amp-story-auto-ads.md]: Done.

Comment (calebcordry) [reply extensions/amp-story/amp-story-auto-ads.md]: ok really done :)

File: extensions/amp-story/amp-story-auto-ads.md [hunk 3]
```diff
@@ -75,45 +72,27 @@ looks like the following:
...
-    </script>
+It has to be a direct child of `amp-story` element.
 
-    <template type="amp-mustache" id="template-1">
-      <amp-img layout="fill" src="{{imgSrc}}"></amp-img>
-      <amp-pixel src="{{impressionUrl}}"></amp-pixel>
-    </template>
+## Insertion Control
+If there is a specific position in a story that you wish to never show an ad,
+you can add the `next-page-no-ad` attribute an `<amp-story-page>`. The insertion
```
Comment (torch2424) [inline extensions/amp-story/amp-story-auto-ads.md]: 🔥

**Pull request merged**